charts/wire-server-metrics/values.yaml

```diff
@@ -0,0 +1,79 @@
+prometheus-operator:
+  prometheus:
+    additionalServiceMonitors:
+      - name: wire-services
+        # We copy these labels from the pod onto the collected metrics from that pod
+        targetLabels:
+          - wireService
+        endpoints:
+          - path: '/i/metrics'
+            port: http
+            interval: 3m
+        selector:
+          matchExpressions:
+            # select any pod with a 'wireService' label
+            - key: wireService
+              operator: Exists
+
+  grafana:
+    adminPassword: "admin"
+    ingress:
+      enabled: false
+    persistence:
+      storageClassName: "aws-ebs-retained"
+      enabled: true
+      accessModes: ["ReadWriteOnce"]
+      size: 10Gi
+
+    dashboardProviders:
+      dashboardproviders.yaml:
+        apiVersion: 1
+        providers:
+          - name: 'default'
+            orgId: 1
+            folder: ''
+            type: file
+            disableDeletion: false
+            editable: true
+            options:
+              path: /var/lib/grafana/dashboards/default
+
+    # TODO: Chart importing is broken for the time being; should be fixed in 
```

Comment:
I created an issue which references this; I'd prefer to merge through without dashboard importing for now since it'll be easiest to wait a week or two for the fix to make it downstream.